supervisor/plugins/observer.py

```diff
@@ -90,6 +91,10 @@ async def start(self) -> None:
             _LOGGER.error("Can't start observer plugin")
             raise ObserverError() from err
 
+    async def stop(self) -> None:
```

Comment:
Is this new functionality? :thinking:

Reply:
Not exactly. It was in every plugin except observer because Supervisor cannot stop observer, it can only restart it. But we were using it like it was in the `BasePlugin` class elsewhere. So I refactored to put it in `BasePlugin` and made the observer override raise a `RuntimeError` saying it was not implemented in that particular plugin.

Reply:
Ok, yeah make sense then!